### ISSUE TITLE
Cannot remove reference prefix from theorem

### ISSUE THREAD

**Issue by Flarp** [RyotaUshio/obsidian-math-booster#207]
I've been using this plugin for a while now, and with 2.0.x coming out, the removal of the project feature has put back the prefixes on theorem references (the note title, along with the greater than symbol). I like to include theorems that are used in proofs, and name them in plain English such that they flow with the sentence when linked (I'll admit this is a weird use-case, but it's worked for me). I've tried using various settings to remove it (such as disabling "Show note title at link's head," but this only seems to disable it for when the theorem is contained within the same note: not when it is referencing a different note). Is it possible to remove this prefix as of the 2.0.x release? I apologize if there is an option I missed that does this; I did look around for a little bit but did not find anything myself that seemed related to this.

Here's a screenshot of an example of how this looks after 2.0.x.

![image](https://github.com/RyotaUshio/obsidian-math-booster/assets/11673881/d54344dd-d1da-4ca4-a40c-0f7fd7cd8653)

**Comment by RyotaUshio** [comment]
Thank you for reporting. I must admit this is my fault; I forgot to take the "Show note title at link's head" setting into consideration. In v2, the code used in v1 was almost completely rewritten, so there may be more mistakes of this sort.

To answer your question, 

>  Is it possible to remove this prefix as of the 2.0.x release?

currenly, no. But I will soon fix the problem that "Show note title at link's head" has no effect.

Also, in the longer term, I'm planning to re-implement the project feature. Let me know if you have any ideas to improve it, e.g. introducing tag-based projects etc.

**Comment by RyotaUshio** [comment]
A fix for the "Show note title at link's head" bug was released as [2.0.3](https://github.com/RyotaUshio/obsidian-math-booster/releases/tag/2.0.3).
